servers/rendering/renderer_rd/forward_clustered/scene_shader_forward_clustered.cpp

```diff
@@ -131,8 +131,12 @@ void SceneShaderForwardClustered::ShaderData::set_code(const String &p_code) {
 
 	SceneShaderForwardClustered *shader_singleton = (SceneShaderForwardClustered *)SceneShaderForwardClustered::singleton;
 	Error err = shader_singleton->compiler.compile(RS::SHADER_SPATIAL, code, &actions, path, gen_code);
-	if (err != OK) {
-		return;
+	if (RS::get_singleton()->is_shader_errors_suppressed()) {
+		if (err != OK) {
+			return;
+		}
+	} else {
+		ERR_FAIL_COND(err != OK);
 	}
```

Comment:
Let's check the less likely first to avoid an unneeded boolean check:
```suggestion
	if (err != OK) {
	    if (RS::get_singleton()->is_shader_errors_suppressed()) {
			return;
		}
		ERR_FAIL_MSG("Shader compilation failed.");
	}
```

Reply:
Done